applications/common/frontend/src/global-variables.scss

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2023 Dynamic Solutions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'src/atomic-design-system/00-token/t.border' as borders;
+@use 'src/atomic-design-system/00-token/t.breakpoint' as breakpoints;
+@use 'src/atomic-design-system/00-token/t.color' as colors;
+@use 'src/atomic-design-system/00-token/t.font' as fonts;
+@use 'src/atomic-design-system/00-token/t.position' as positions;
+@use 'src/bulma-overwrite/variables' as variables;
+
+@mixin add-css-variables($module, $short-name) {  
```

Comment:
let's rename this variable: $module -> $map because in SASS you pass a map, not a module.

Reply:
done